pkg/virt-controller/network/multus_annotations.go

```diff
@@ -114,18 +119,19 @@ func newBindingPluginMultusAnnotationData(kvConfig *v1.KubeVirtConfiguration, pl
 	}, nil
 }
 
-func newMultusAnnotationData(namespace string, interfaces []v1.Interface, network v1.Network, podInterfaceName string) networkv1.NetworkSelectionElement {
+func newMultusAnnotationData(namespace string, interfaces []v1.Interface, network v1.Network, podInterfaceName string, ipamClaimReference string) networkv1.NetworkSelectionElement {
```

Comment:
I really don't like the fact you're adding this attribute here. 

Would it make sense to add another func for this ? Something like `newMultusAnnotationDataWithIPAMClaim` ? 

```golang
func newMultusAnnotationDataWithIPAMClaim(namespace string, interfaces []v1.Interface, network v1.Network, podInterfaceName string, ipamClaimName string) networkv1.NetworkSelectionElement {
    networkSelectionElement := newMultusAnnotationData(...)
    networkSelectionElement.IPAMClaimReference = ipamClaimName
    return networkSelectionElement
}


func newMultusAnnotationData(namespace string, interfaces []v1.Interface, network v1.Network, podInterfaceName string) networkv1.NetworkSelectionElement {
	multusIface := vmispec.LookupInterfaceByName(interfaces, network.Name)
	namespace, networkName := getNamespaceAndNetworkName(namespace, network.Multus.NetworkName)
	var multusIfaceMac string
	if multusIface != nil {
		multusIfaceMac = multusIface.MacAddress
	}
	return networkv1.NetworkSelectionElement{
		InterfaceRequest: podInterfaceName,
		MacRequest:       multusIfaceMac,
		Namespace:        namespace,
		Name:             networkName,
	}
}
```